.github/workflows/node-version-up.yml

```diff
@@ -0,0 +1,63 @@
+name: Node Version Up
+
+on:
+  workflow_dispatch:
+    inputs:
+      versioning:
+        description: "node version up: major, minor, or patch"
```

Comment:
Does this allow setting a custom version? If I want to set the version to `1.1.0-beta`, would I be able to pass this here and it's applied like that in the Cargo.toml?

Reply:
So, cargo edit allows 6 options: [major, minor, patch, release, rc, alpha, beta](https://github.com/killercup/cargo-edit/blob/f7ad38c74c03e9cb6dab3eeb82601a88ef2f2c06/src/bin/set-version/version.rs#L60-L67). 

Example of choosing alpha: `version = "0.1.3-alpha.1"`

Reply:
That's nice 👍 

I think there should also be the option to set an [absolute version](https://github.com/killercup/cargo-edit/blob/f7ad38c74c03e9cb6dab3eeb82601a88ef2f2c06/src/bin/set-version/version.rs#L30) instead of bumping it. I guess we'd need to use that more often, e.g. looking at the next bump from v1.1.0 to v1.6.0, we can't use any of the relative bumps as we want to skip a couple versions at once. I think in that case we can just do it with `cargo set-version 1.6.0` ie without the `--bump` parameter.

Reply:
Can't do it in `set-version`. I've copied [the script from spacewalk](https://github.com/pendulum-chain/spacewalk/blob/main/.maintain/create_minor_release.sh) that requires a _version_ argument.
example: `.github/scripts/specific_version_up 1.6.0`